### DISCUSSION TITLE
Discussions

### DISCUSSION THREAD

**Discussion by annie444** [Sirneij/rust-auth#12] [Announcements]
Hi Sirnejj,

I'm sorry for opening an issue (maybe turn on discussions for the repo?), but I've been following your work here as I'm implementing similar protocols for my business (Rust backend on a Sveltekit frontend), and I find your work incredible! 

I've been thinking of extending this to use the Deno integration through yum, and then serving the SvelteKit site on deno. I don't know how well it would work, but though you might be interested!

I also have redone my version to use SeaORM and TypeShare (by 1Password FOSS) to allow full type-safety between my typescript front end and rust backend. 

Overall amazing work! I would love to collaborate if you're wanting :)

**Comment by Sirneij** [comment]
Hello Annie,

It seems that your plan is awesome and I am willing to collaborate.

Between, I have turned on "discussions" for this repo. I really appreciate your compliments.

**Reply by annie444** [reply]
So I found an issue with the Tcp listener, and was able to resolve the issue by not "starting" (ref startup.rs) a generic TCP server, but rather passing settings.application.{host, port} to the actin_web HttpServer .bind() method as a tuple. 

The generic TCP listener can't handle middleware binding and was causing all of the issues. (CORS works now).

**Reply by Sirneij** [reply]
Great!
